encrypt_credentials.py

```diff
@@ -6,8 +6,10 @@
 import jinja2
 import os
 from ansible.parsing.vault import VaultEditor, VaultSecret, is_encrypted
+from openpyxl import load_workbook
```

Comment:
Does this require changes to setup.sh file?

Reply:
No, openpyxl is already in pip_requirements.txt